.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: python
 
-python:
-  - "2.7"
+# python:
+  # - "2.7"
   #- "3.5"
```

Comment:
Probably makes sense to update to 3.8 :)

Reply:
So far relying on whatever TravisCI suggests out of the box. Will need to revisit later with all thee versions for all dependencies...